module/Application/view/layout/layout.phtml

```diff
@@ -111,21 +142,25 @@ function getIsActiveClass($view, array $condition)
             <?php echo $this->content; ?>
             <hr>
             <footer class="row">
-                <div class="col-md-2 col-sm-4">
+                <div class="col-md-3 col-sm-3">
                     <img src="<?= $this->basePath() ?>/img/logo-white.png" />
                 </div>
-                <div class="col-md-4 col-sm-8">
+                <div class="col-md-4 col-sm-4">
                     <h3>Studievereniging GEWIS</h3>
                     <address>
                         MF 3.155<br>
-                        De Groene Loper 5<br>
-                        5612AZ Eindhoven<br>
+                        Postbus 513<br>
```

Comment:
Dit is nu een beetje verwarrend. Postbus 513 is het post adres, MF 3.155 is het lokaal in het bezoekadres. Dit hoort niet bij elkaar te staan. 

Ik stel voor dit gewoon hetzelfde te doen als op de oude site, en de twee dingen te scheiden. Of is er een reden voor dat alleen dit adres gegeven wordt?